src/output-json-alert.c

```diff
@@ -576,7 +576,7 @@ static void AlertAddFiles(const Packet *p, JsonBuilder *jb, const uint64_t tx_id
                     jb_open_array(jb, "files");
                 }
                 jb_start_object(jb);
-                EveFileInfo(jb, file, file->flags & FILE_STORED);
+                EveFileInfo(jb, file, file->flags & FILE_STORE);
```

Comment:
this is not correct. FILE_STORE is set if the file should be stored, but only the file store module sets FILE_STORED if the file has actually been stored.